tools/base/fpgad/fpgad.service.in

```diff
@@ -0,0 +1,21 @@
+# Intel FPGA daemon
+[Unit]
+Description = FPGA monitor
+
+[Service]
+Type=simple
+StandardOutput=journal+console
+StandardError=journal+console
+EnvironmentFile=-/etc/sysconfig/fpgad.conf
```

Comment:
I'm not familiar with the syntax of the service config file. But is the **=-** intentional and correct for EnvironmentFile?

Reply:
The '-' after the '=' says to ignore the error if the file does not exist.